pr2eus/robot-interface.l

```diff
@@ -1284,9 +1284,11 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
   (:go-stop (&optional (force-stop t))
    (when joint-action-enable
      (send move-base-action :cancel-all-goals)
-     (prog1
-         (send move-base-trajectory-action :cancel-all-goals)
-       (if force-stop (send self :go-velocity 0 0 0)))
+     (when move-base-trajectory-action
+       (prog1
+           (send move-base-trajectory-action :cancel-all-goals)
+           (if force-stop (send self :go-velocity 0 0 0)))
```

Comment:
use indentation with 2 spaces
```
(prog1
  (send move-base-trajectory-action :cancel-all-goals)
  (if force-stop (send self :go-velocity 0 0 0)))
```

Reply:
I think this is default indent. force pushed with fixed version
```
     (when move-base-trajectory-action
       (prog1
           (send move-base-trajectory-action :cancel-all-goals)
         (if force-stop (send self :go-velocity 0 0 0)))

```